src/backend/latex/preamble.rs

```diff
@@ -141,6 +141,31 @@ pub fn write_fixes(cfg: &Config, out: &mut impl Write) -> Result<()> {
     Ok(())
 }
 
+pub fn write_deep_headers(cfg: &Config, out: &mut impl Write) -> Result<()> {
+    fn write_simple_header(name: &str, into: &mut impl Write,  font: &str, level: i32) -> Result<()> {
+        writeln!(into, "{}", "\\makeatletter")?;
+        write!(into, r#"\newcommand\{}"#, name)?;
+        write!(into, "{}{}{}{}", r#"{\@startsection{"#, name, "}{", level)?;
```

Comment:
You can have `{`/`}` in the result of format strings by using double `{{`/`}}` like so: `println!("{{{}}}", 1337) // "{1337}"`.